src/frontend/data/sideEffects/pollForTrack/index.tsx

```diff
@@ -35,7 +35,7 @@ export async function pollForTrack<
       ? Document
```

Comment:
_in commit title_
remove *duplicate* code

_in commit message_
we *didn't notice*